tests/helper.py

```diff
@@ -1,8 +1,8 @@
 from errbot.backends.test import TestBot
 
-def plugin_testbot(klass, loglevel, config=None):
+def plugin_testbot(klass, plugin_name, loglevel, config=None):
     config = config if config else dict()
     testbot = TestBot(loglevel=loglevel, extra_config=config)
     testbot.start()
-    plug = testbot.bot.plugin_manager.instanciateElement(klass)
+    plug = klass(testbot.bot, plugin_name)
```

Comment:
```
def instanciateElement(self, element) -> BotPlugin:
    """Overrides the instanciation of plugins to inject the bot reference."""
    return element(self.bot, name=self._current_pluginfo.name)
```

Reply:
@meetmangukiya @gbin does it make a difference, where a class gets instantiated?

Reply:
No this is not the cause for failures. Facing the `TemplateNotFound` exception due to the changes made to `errbot.templating` by this commit https://github.com/errbotio/errbot/commit/9a03b72a33c12a34c037c0f4286d5192c3f766b6#diff-b27e6e3f6d8b53f810de95de950feea9

Reply:
Figured a workaround, explore yourself further:

```
From d9858f2dc7162273d8315ae37bfee6467257599e Mon Sep 17 00:00:00 2001
From: Meet Mangukiya <meetmangukiya98@gmail.com>
Date: Wed, 11 Jul 2018 06:44:23 +0530
Subject: [PATCH] Fix templating issue

---
 tests/labhub_test.py | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/tests/labhub_test.py b/tests/labhub_test.py
index 9d69a64..e06b195 100644
--- a/tests/labhub_test.py
+++ b/tests/labhub_test.py
@@ -19,9 +19,21 @@ from plugins.labhub import LabHub
 
 from tests.helper import plugin_testbot
 
+
+import errbot
+from errbot.plugin_info import PluginInfo
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
+
+
 class TestLabHub(unittest.TestCase):
 
     def setUp(self):
+        labhub_info_path = Path(__file__).parent / '..' / 'plugins' / 'labhub.plug'
+        with labhub_info_path.open() as plugfile:
+            labhub_info = PluginInfo.load_file(plugfile, labhub_info_path)
+        add_plugin_templates_path(labhub_info)
+
         plugins.labhub.github3 = create_autospec(github3)
 
         self.mock_org = create_autospec(github3.orgs.Organization)
-- 
2.18.0
```